tests/unit/reports/test_utils.py

```diff
@@ -1189,3 +1191,127 @@ def test_aggregate_metric_results():
     # Assert
     assert avg_score == 0.45
     assert num_errors == 1
+
+
+def test__validate_categorical_values():
+    """Test no extra categoricals does not crash."""
+    # Setup
+    sdtype = 'categorical'
+    real_data = pd.DataFrame({'col1': [1, 2, 3, 4]})
+    synthetic_data = pd.DataFrame({'col1': [1, 2, 4, 4]})
+    metadata = {'fields': {'col1': {'type': sdtype}}}
+    warnings.filterwarnings('error', category=UserWarning)
```

Comment:
do we need to filter and reset warnings for this one? My understanding is that none are getting raised here and it doesn't look like we're checking them

Reply:
Yeah, my intention here was to test that the warning doesn't get raised when there aren't extra categories. Since it's just a warning, the only way to get the test to fail if the warning occurs is to have the warning to error.